.github/workflows/e2e_ui_test.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: FoodStandardsAgency/register-a-food-business-UI-tests
-          ref: "master"
+          ref: "selenium-test-update"
```

Comment:
is this supposed to remain like that?

Reply:
This one has been changed to a GitHub Actions variable https://github.com/FoodStandardsAgency/register-a-food-business-front-end/pull/416/commits/8fc7bb7577fc02cab3ed658976429161ec0226b8